### DISCUSSION TITLE
Video: check/edit mesh, abl, with dial gauge holder, mriscoc firmware, sprite extruder

### DISCUSSION THREAD

**Discussion by hayden-t** [mriscoc/Ender3V2S1#283] [Show and tell]
https://youtu.be/NlNbXGhCbRE

This model I built will allow you to check your bl/crtouch probe mesh for accuracy, it seems the gap between probe location and nozzle location leads to inaccuracy.

**Comment by mriscoc** [comment]
Great video, I want to see the results from using the special support for the CRTouch.
